src/questions/aave.yaml

```diff
@@ -0,0 +1,243 @@
+  difficultyLevel: "Medium"
+  choices:
+    - content: Interest on supplied asset
+      key: A
+      
+    - content: Collateral trading and repaying with collateral
+      key: B
+      
+    - content: Debt tokenization
+      key: C
+      
+    - content: Credit delegation
+      key: D
+          
+- uuid: a153bd00-916b-4eef-81d4-20fe5be5014a
+  type: SingleChoice
+  content: Which of the following are use cases of aTokens
+  hint: NoHint
+  explanation: aTokens are used to accrue interest, and they can also be used to directly repay loans without redeeming them for their underlying tokens.
+  answerKeys:
+    - D
+  difficultyLevel: "Medium"
+  choices:
+    - content: Repay loans
+      key: A
+      
+    - content: Accrue interest on supplied collateral
+      key: B
+      
+    - content: Debt tokenization
+      key: C
+      
+    - content: Both A and B
+      key: D
+
+- uuid: 048e1805-d768-4988-804d-617a7b3ed51d
+  type: SingleChoice
+  content: Which of the following is a use case for E-Mode
+  hint: NoHint
+  explanation: E-Mode allows the borrower to maximize capital efficiency by increasing the LTV and lowering the liquidation threshold on assets borrowed in E-Mode.
+  answerKeys:
+    - B
+  difficultyLevel: "Medium"
+  choices:
+    - content: Minimize capital efficiency
+      key: A
+      
+    - content: Maximize capital efficiency
+      key: B
+      
+    - content: To borrow newly listed assets
+      key: C
+      
+    - content: To improve the security of the protocol
+      key: D
+            
+- uuid: 7e5100a4-5aa8-423c-a38d-8c1c53f72ded
+  type: SingleChoice
+  content: Which of the following modes in Aave allows the user to list new assets with a designated debt ceiling?
+  hint: NoHint
+  explanation: Isolation mode on Aave lets you list new assets as Isolated with a designated debt ceiling. 
+  answerKeys:
+    - A
+  difficultyLevel: "Medium"
+  choices:
+    - content: Isolation mode
+      key: A
+      
+    - content: E-Mode
+      key: B
+      
+    - content: Siloed borrowing mode
+      key: C
+      
+    - content: New listing mode
+      key: D
+
+- uuid: 982e2c16-d5b7-473d-ada9-605f35e14794
+  type: SingleChoice
+  content: In what denomination are the rewards users get for staking Aave in the Safety Module?
+  hint: NoHint
+  explanation: The Aave Protocol V3 now supports multiple rewards per token, which means that it's possible for an asset listing to enable additional incentive rewards denominated in underlying tokens. 
+
+  answerKeys:
+    - D
+  difficultyLevel: "Medium"
+  choices:
+    - content: In stkAAVE
+      key: A
+      
+    - content: In underlying tokens
+      key: B
+      
+    - content: Only in aTokens
+      key: C
+      
+    - content: Both A and B
+      key: D
+      
+
+
+- uuid: 9e2e7ef6-2978-47bc-a6c9-29c38d686436
+  type: SingleChoice
+  content: Debt tokens are transferrable.
+  hint: NoHint
+  explanation: False, debt tokens are not transferrable.
```

Comment:
What happens when someone transfers the debt tokens?

Reply:
Although debt tokens are modelled on the ERC20/EIP20 standard, they are non-transferrable. Therefore they do not implement any of the standard ERC20/EIP20 functions relating to transfer() and allowance().